react/components/AutocompleteInput/Option.tsx

```diff
@@ -72,7 +70,6 @@ const Option: React.FunctionComponent<
 
   return (
     <button
-      key={key}
```

Comment:
No chance if this breaking anything?

Reply:
No, key should never be read from props, the behavior in that case is not specified (it was just undefined).